iron-overlay-behavior.html

```diff
@@ -118,6 +119,14 @@
         type: Object
       },
 
+      /**
+       * The HTMLElement that will be firing relevant KeyboardEvents.
+       */
+      keyEventTarget: {
```

Comment:
Is this used anywhere? Or is it just to override the `IronA11yKeysBehavior` default? If the latter, maybe add that in the comment ("overriden from x") -- otherwise a future keen party might delete it by accident 😂